Comments/Models/Context.cs

```diff
@@ -1011,5 +1011,55 @@ public bool AreAnswersForThisOrganisation(List<int> answerIds, int organisationI
 				.Any(c => c.OrganisationUser.OrganisationAuthorisation.OrganisationId.Equals(organisationId));
 
 		}
-	}
+
+        public int CountCommentsAndAnswerSubmissionsForThisOrganisation(string sourceURI, int organisationId)
+        {
+            var comments = Comment
+                .Include(c => c.OrganisationUser)
+                    .ThenInclude(ou => ou.OrganisationAuthorisation)
+                .IgnoreQueryFilters()
+                .Where(c => (
+                            (c.Location.SourceURI.Contains($"{sourceURI}/") || c.Location.SourceURI.Equals(sourceURI))
+                            && c.StatusId == (int)StatusName.SubmittedToLead)
+                            )
+                .ToList();
+
+            var commentsCount = comments.Where(c => c.OrganisationUser?.OrganisationAuthorisation != null)
```

Comment:
do you think the count could be returned straight from the database please? 

same with the answers line 1040.